src/simtools/runners/runner_services.py

```diff
@@ -191,7 +191,7 @@ def _get_data_file_path(self, file_type, file_name, run_number):
             "output": ".zst",
             "corsika_output": ".corsika.zst",
             "simtel_output": ".simtel.zst",
-            "event_data": ".hdf5",
+            "event_data": ".event_data.hdf5",
```

Comment:
Should this name be discussed? Are we happy with it? If it only contains information about triggered telescopes, why not include trigger in the name? Something like `trigger_data`, `trigger_events`, `trig_evt`, `triggered_telescopes`, `trig_tel`, etc.?

Reply:
And I am not happy about it, but since weeks we cannot come up with something better.

Reply:
I think we are using somewhere 'reduced' in the code already. Would `reduced_event_data` be too long?

Reply:
No, let's go for `reduced_event_data`. There is a limit on the grid for the length of file names, but let's hope we are not at that limit that six more characters get us there.

Reply:
Changed it (and changing file names is really a bit of a nightmare, as one has to find it in several places in the code...)